operator/src/main/resources/scripts/initializeDomainHomeOnPV.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# Copyright (c) 2023, Oracle and/or its affiliates.
+
+scriptDir="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+. ${scriptDir}/utils_base.sh
+[ $? -ne 0 ] && echo "[SEVERE] Missing file ${scriptDir}/utils_base.sh" && exit 1
+UNKNOWN_SHELL=true
+
+checkEnv DOMAIN_HOME || exit 1
+
+ORIGIFS=$IFS
+trace "DOMAIN HOME is " $DOMAIN_HOME
```

Comment:
Administrators deal with 'spec.domainHome' and it helps to put values in single quotes, so: 
"'spec.domainHome' is '$DOMAIN_HOME'.".